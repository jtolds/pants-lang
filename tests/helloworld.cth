hello := { |target|
  print "Hello" (+ target "!")
}

hello "world"
