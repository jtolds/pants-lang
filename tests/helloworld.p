hello = { |target|
  println "Hello" (+ target "!")
}

hello "world"
