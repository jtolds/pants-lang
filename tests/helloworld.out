Hello world!

return code: 0
